src/App.js

```diff
@@ -1,9 +1,106 @@
+import { useState } from "react";
+import './style.css'; //이번 과제는 컴포넌트 분리가 어려워서 css를 사용했습니다 ㅠㅠ
+import './background.js';
+import Clock from './clock.js';
 
 function App() {
+  const [toDo, inputToDo] = useState("");
+  const [toDos, setToDos] = useState([]);
+  const [doneToDos, movetoDos] = useState([]); 
+
+  //inputToDo를 통해 입력받음
+  const onChange = (event) => inputToDo(event.target.value);
+
+
+  const submitInput = (event) => {
+      event.preventDefault();
+      if (toDo === "") {
+        return;
+      }
+      setToDos((currentArray) => [toDo, ...currentArray]); //입력받은 걸 배열에 넣음
+      inputToDo("");
+
+    };
+
+    const moveDoneButton = (item,index) =>{
+    
+      movetoDos((current) => [item, ...current]); //Done 리스트의 배열로 추가됨
```

Comment:
이 부분도 이름을 명확하게 해 주시는 게 좋습니다

```suggestion
      setDoneToDoList((doneToDoList) => [item, ...doneToDoList]); //Done 리스트의 배열로 추가됨
```

Reply:
훨씬 명확하네요,,,! 감사합니다 ㅠ.ㅠ